src/main/scala/system/RocketTestSuite.scala

```diff
@@ -119,14 +119,14 @@ object DefaultTestSuites {
   val rv32siNames = LinkedHashSet("csr", "ma_fetch", "scall", "sbreak", "wfi", "dirty")
   val rv32si = new AssemblyTestSuite("rv32si", rv32siNames)(_)
 
-  val rv32miNames = LinkedHashSet("csr", "mcsr", "illegal", "ma_addr", "ma_fetch", "sbreak", "scall")
+  val rv32miNames = LinkedHashSet("csr", "mcsr", "illegal", "ma_addr", "ma_fetch", "sbreak", "scall", "breakpoint", "lh-misaligned", "lw-misaligned", "shamt", "sh-misaligned", "sw-misaligned", "zicntr")
   val rv32mi = new AssemblyTestSuite("rv32mi", rv32miNames)(_)
 
   val rv32u = List(rv32ui, rv32um)
   val rv32i = List(rv32ui, rv32si, rv32mi)
   val rv32pi = List(rv32ui, rv32mi)
 
-  val rv64uiNames = LinkedHashSet("addw", "addiw", "ld", "lwu", "sd", "slliw", "sllw", "sltiu", "sltu", "sraiw", "sraw", "srliw", "srlw", "subw")
+  val rv64uiNames = LinkedHashSet("addw", "addiw", "ld", "lwu", "sd", "slliw", "sllw", "sltiu", "sltu", "sraiw", "sraw", "srliw", "srlw", "subw", "ma_data")
```

Comment:
It seems that ma_data for rv64ui-v will [hang](https://github.com/riscv-software-src/riscv-tests/issues/419), at least for the mill part

Reply:
You are right, those should be a in a separate suite